invenio_records_resources/services/files/tasks.py

```diff
@@ -19,17 +19,20 @@
 def fetch_file(service_id, record_id, file_key):
     """Fetch file from external storage."""
     service = current_service_registry.get(service_id)
+    # FIXME HERE
```

Comment:
i'm not really sure how to handle the errors in the tasks... should we put a `try/except` around and log the error, or should we just simply ignore the exception?

Reply:
I would definitely log the errors, so it can be monitored. A general `try/except` around the task might be enough, imo.

In general and just as an idea: To not duplicate it in every task, how about introducing a general task error handler, or sub-classing the celery task and define an error handler? Then this task could be used almost everywhere?

Reply:
I've added the `try/except` with logging in the tasks themselves for now, and I'd create a follow-up issue for the improvement suggestion.
Where do you think it would be most appropriate?